js/README.md

```diff
@@ -0,0 +1,28 @@
+# Frontend apps
+
+## Trivial app
+
+Consider using only static files (example: [Green Card Predictor](https://github.com/renuo/green-card-probability-frontend))
+and CDNs.
+
+## Simple JS app
+
+If you need a really simple JavaScript application and you need more than just an *index.html*, you can checkout this template:
+https://github.com/renuo/tamedia-altersheime/tree/0a5fc447bc7fdae9b81bbe0918ca64c864c58354
```

Comment:
👍  I cannot go too much in detail in the template. I'd like if also @macav could take a look at it. 
I am just wondering if, for such a simple app, cucumber is not an overkill.. 🤔

Reply:
I think cucumber is the easiest way to have tests at all. I think unit tests would be overkill.